### DISCUSSION TITLE
LilyGO Esp 32 with Datacake

### DISCUSSION THREAD

**Discussion by dsandels** [cyberman54/ESP32-Paxcounter#979] [General]
Hi,

We just compiled and loaded the PAX code to get the LilyGO working. The load was successful and the LilyGO LCD shows the Wi-Fi count and no BLE count at the moment. My question is related to using the ESP decoder out on Datacake. Instead of getting the actual Wi-Fi count on the LilyGO board, I get a much high count over the Helium LoRaWAN. What could possibly be wrong ?
I have also set Cyclic Unconfirmed Counting.

I have attached a photo of the LilyGO board, and three screen shots of the Decoder results. Thanks !

![IMG_6993](https://github.com/cyberman54/ESP32-Paxcounter/assets/64983909/86c475c1-2bd6-456e-8090-73d4bc910dce)
![IMG_6992](https://github.com/cyberman54/ESP32-Paxcounter/assets/64983909/2deda16d-81c9-4f0b-9822-93e46c6b11c9)
![IMG_6991](https://github.com/cyberman54/ESP32-Paxcounter/assets/64983909/6ef783e7-2a95-4557-bf5f-dcd9b31c21f0)
![IMG_6990](https://github.com/cyberman54/ESP32-Paxcounter/assets/64983909/7c1c19b8-9dae-44d7-b7ca-9fc6298c1bb3)

**Comment by cyberman54** [comment]
This looks like a decoder logic issue. Thus, you should doublecheck your payload decoders in Datacake, i.e. check the byte-endianess.

**Comment by cyberman54** [comment]
I'm closing this, because [Datacake](https://datacake.co/solutions/datacake-lorawan-people-counter-paxcounter-personenzaehler) seems a commercial project to me, which revenues are based on this open source software. Since Datacake never has contributed to this project, they shall take care of their customer support.

Please contact Datacake for support.
